packages/core/src/shortref.ts

```diff
@@ -30,9 +30,7 @@ export interface ShortRef extends Doc {
 export async function createShortRef<T extends Doc> (
   storage: Storage,
   user: Ref<Account>,
-  space: Ref<Space>,
```

Comment:
I'm not very like this changes, not sure why.

Reply:
Only about "space" field or all of this?

Reply:
If you like just to simplify passing 3 params instead of 1 param, let's create an interface with just 3 this params, in this case you will be able to pass entire doc, but still will be possible to call this method without passing a doc, but just required parameters.